src/Uno.UWP/Devices/Radios/Radio.Android.cs

```diff
@@ -0,0 +1,312 @@
+				{
+					typeof(Radio).Log().Debug("GetRadiosWiFiOrCellular(" + radioKind.ToString() + "): no 'AccessNetworkState' permission, check app Manifest");
+				}
+				radio.State = RadioState.Unknown;
+				return radio;
+			}
+
+			var sysService = context.GetSystemService(Android.Content.Context.ConnectivityService);
+			if (sysService is null)
+			{
+				if (typeof(Radio).Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Warning))
+				{
+					typeof(Radio).Log().Debug("GetRadiosWiFiOrCellular(" + radioKind.ToString() + "): GetSystemService returns null, assuming no radio");
+				}
+				return null;
+			}
+
+			var connManager = (Android.Net.ConnectivityManager)sysService;
+			if (connManager is null)
+			{
+				if (typeof(Radio).Log().IsEnabled(Microsoft.Extensions.Logging.LogLevel.Warning))
+				{
+					typeof(Radio).Log().Debug("GetRadiosWiFiOrCellular(" + radioKind.ToString() + "): Android.Net.ConnectivityManager is null, assuming no WiFi radio");
+				}
+				return null;
+			}
+
+			if (Android.OS.Build.VERSION.SdkInt > Android.OS.BuildVersionCodes.LollipopMr1)
+			{
+				// since API 23
+				var radiostate = GetRadioStateForNet(connManager, radioKind);
+				if (radiostate is null || !radiostate.HasValue)
+				{
+					// detailed Log entry already done in GetRadioStateForNet
+					return null;
+				}
+
+				if (radiostate.Value)
+				{
+					radio.State = RadioState.On;
+				}
+				else
+				{
+					radio.State = RadioState.Off;
+				}
+
+				return radio;
+			}
+			else
+			{
+				// for Android API 1 to 28 (deprecated in 29)
+#pragma warning disable CS0618 // Type or member is obsolete
+				var netInfo = connManager.ActiveNetworkInfo;
+				if (netInfo is null)
+				{
+					radio.State = RadioState.Off;
+				}
+				else
+				{
+					radio.State = RadioState.Off;
+					if (radioKind == RadioKind.WiFi)
+					{
+						if (netInfo.Type == Android.Net.ConnectivityType.Wifi ||
+							netInfo.Type == Android.Net.ConnectivityType.Wimax)
+						{
+							radio.State = RadioState.On;
+						}
+					}
+					else
+					{
+						if (netInfo.Type == Android.Net.ConnectivityType.Mobile)
+						{
+							radio.State = RadioState.On;
+						}
+					}
+#pragma warning restore CS0618 // Type or member is obsolete
+
+				}
+				return radio;
+			}
+		}
+
+
+		private async static Task<IReadOnlyList<Radio>> GetRadiosAsyncTask()
+		{
+			var radios = new List<Radio>();
+
+			var radio = GetRadiosBluetooth();
+			if (radio != null) radios.Add(radio); // yield oRadio;
+
+			radio = GetRadiosWiFiOrCellular(RadioKind.WiFi);
+			if (radio != null) radios.Add(radio); // yield oRadio;
+
+			radio = GetRadiosWiFiOrCellular(RadioKind.MobileBroadband);
+			if (radio != null) radios.Add(radio); // yield oRadio;
+
+			return radios;
+		}
+
+		/// <summary>
+		/// Gets info about radio devices which exist on the system
+		/// </summary>
+		public static IAsyncOperation<IReadOnlyList<Radio>> GetRadiosAsync()
+		=> GetRadiosAsyncTask().AsAsyncOperation();

```

Comment:
Indentation

Reply:
I applied ^K ^D (Edit:Advanced:Format document), nothing changes. So, I added Tab manually, I hope it would be sufficient.